apps/builder/app/builder/features/sidebar-left/panels/marketplace/card.tsx

```diff
@@ -0,0 +1,120 @@
+import {
+  Flex,
+  Text,
+  theme,
+  focusRingStyle,
+  css,
+  rawTheme,
+} from "@webstudio-is/design-system";
+import env from "~/shared/env";
+import { Image, createImageLoader } from "@webstudio-is/image";
+import { forwardRef } from "react";
+import { SpinnerIcon } from "@webstudio-is/icons";
+
+const focusOutline = focusRingStyle();
+
+const imageLoader = createImageLoader({
+  imageBaseUrl: env.IMAGE_BASE_URL,
+});
+
+const imageContainerStyle = css({
+  position: "relative",
+  overflow: "hidden",
+  aspectRatio: "1.91",
+});
+
+const spinnerStyle = css({
+  position: "absolute",
+  inset: 0,
+  display: "flex",
+  alignItems: "center",
+  justifyContent: "center",
+  backgroundColor: "rgba(255, 255, 255, 0.5)",
+  backdropFilter: "blur(8px)",
+});
+
+const imageStyle = css({
+  position: "absolute",
+  top: 0,
+  width: "100%",
+  height: "100%",
+  objectFit: "cover",
+  transition: "transform 100ms",
+  "&:hover": {
+    transform: "scale(1.1)",
+  },
+});
+
+type ThumbnailProps = {
+  image?: { name: string } | string | "";
```

Comment:
In what case we can have image as string? 
`| ""` makes no sense as covered by string type

Reply:
the Card is used for page templates and in pages its we either take socialImageUrl or socialImageAsset, socialImageUrl is a url or can be an empty string

Reply:
I thought it would be benefitial to see the possibility of an empty string

Reply:
removed it